### DISCUSSION TITLE
[✨] How to use both ssg and ssr for the same route

### DISCUSSION THREAD

**Discussion by enesflow** [QwikDev/qwik-evolution#107] [Proposals For Qwik]
### Is your feature request related to a problem?

I have a dynamic path in my app, `/search/[query]/index.tsx`. I wanted to optimize it by using ssg for some popular search queries like this:
```ts
export const onStaticGenerate: StaticGenerateHandler = async () => {
  const words = ["popular", "search", "queries"];
  return {
    params: words.map((i) => ({ query: i })),
  };
};
```
But when I search for something that is not generated with ssg, I get a 404 error. I want it to fallback to the normal path (`/search/[query]/index.tsx`) and get the data from the server as I've defined it there.

### Describe the solution you'd like

An option to fallback to ssr for not ssg generated paths would be helpful.

### Describe alternatives you've considered

Not using ssg.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
